tests/test_prove.rs

```diff
@@ -102,7 +117,7 @@ fn execute_and_prove_program(
 
     let result = invoke_cli(
         false,
-        &vec![program.as_path()],
+        &[program.as_path()],
```

Comment:
Does the clippy call in the CI not work? Feels like a clippy issue that should have been caught by the CI.